content/en/docs/kitex/FAQ/_index.md

```diff
@@ -33,10 +33,15 @@ description: >
 
 ## Kitex Code Generation Tool
 
-**Q:'not enough arguments' problem when installing the code generation tool**  
+**Q:'not enough arguments' problem when installing the code generation tool**
 * Please try: "go mod：GO111MODULE=on go get github.com/cloudwego/kitex/tool/cmd/kitex@latest"
 
-**Q: Would code generated by new interface overwrite handler.go**  
+**Q: Would code generated by new interface overwrite handler.go**
 * Generated code under kitex_gen/ will be overwritten. But handler.go of server will not be overwritten, only new methods will be added correspondingly.
 
+**Q: Does the template in the current code generation tool support user customization?**
+* At present, there is no plan to support customized templates, because the parameter transfer design will be much more complex. The current plugin mechanism can completely realize any equivalent function
+
+**Q: Whether the "–type" in the code generation tool can be automatically determined by the suffix of the IDL file name and "–module" is changed to default through the module information in go.mod is determined, and use "-module" parameter when multiple files ?**
```

Comment:
* Whether ... can => Can ...
* the "–type" in => the "-type" parameter of
* the suffix of the IDL file name => the filename extension
* and "–module" is changed to default through the module information in go.mod is determined, and use "-module" parameter when multiple files ? => 拆分成另一个问句吧。Can the "-module" parameter use module name from a go.mod it can found by default and require a value only when there is no go.mod or it meets ambiguity?